packages/usa-in-page-navigation/src/test/test-patterns/test-custom-content-selector.twig

```diff
@@ -1,10 +1,5 @@
 <div class="usa-in-page-nav-container">
   <aside class="usa-in-page-nav"
-    data-title-text="On this page"
-    data-title-heading-level="h4"
-    data-scroll-offset="0"
-    data-root-margin="0px 0px 0px 0px"
-    data-threshold="1"
```

Comment:
Removed these attributes so that the component uses defaults - this should improve clarity in testing.